### PR TITLE
Invalidate config in case of wrong issuer

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**
- Invalidate config in case of wrong issuer
- Fix a bug with empty activity view
- Bump version

### CODE REVIEW

File: connect/src/com/telenor/connect/id/ConnectIdService.java [hunk 1]
```diff
@@ -182,13 +192,13 @@ private void clearCookies(Context context) {
         }
     }
 
-    public void logOut(final Context context) {
+    public void logOut(final Context context, final boolean shouldNotify) {
```
Comment (ulfcarlin) [inline connect/src/com/telenor/connect/id/ConnectIdService.java:195]: It seems like this method is always called with shouldNotify=true, so maybe adding the parameter is not needed, neither for logOut nor for revokeTokens?

**Review by ulfcarlin — COMMENTED** [review]
Will all "three changes" in this commit be mentioned in the commit message?

File: connect/src/com/telenor/connect/id/ConnectIdService.java (hunk 1 above)
Comment (SerhiiBovtriuk) [reply connect/src/com/telenor/connect/id/ConnectIdService.java:195]: Yep, that's left from revoke testing, will be fixed.

File: connect/src/com/telenor/connect/WellKnownConfigStore.java [hunk 2]
```diff
@@ -25,6 +26,23 @@ public void set(WellKnownAPI.WellKnownConfig wellKnownConfig) {
                 .apply();
     }
 
+    @SuppressLint("ApplySharedPref")
+    public void cleanSynchronously() {
```
[the remark this answers is not in the document]
Comment (SerhiiBovtriuk) [reply connect/src/com/telenor/connect/WellKnownConfigStore.java:30]: Fixed

File: connect/src/com/telenor/connect/ConnectSdk.java [hunk 3]
```diff
@@ -588,6 +592,21 @@ private static boolean fetchBooleanProperty(ApplicationInfo appInfo, String prop
         return (Boolean) booleanPropertyObject;
     }
 
+    private static void validateWellKnownConfigIdp(Context context) {
+        String wellKnownConfigIssuer = null;
+        if (wellKnownConfig != null) {
+            wellKnownConfigIssuer = wellKnownConfig.getIssuer();
+        }
+        String idpIssuer = ConnectUrlHelper.getConnectApiUrl(idProvider, useStaging) + ConnectUrlHelper.OAUTH_PATH;
+        if (!idpIssuer.equals(wellKnownConfigIssuer)) {
```
[the remark this answers is not in the document]
Comment (SerhiiBovtriuk) [reply connect/src/com/telenor/connect/ConnectSdk.java:601]: Yes, wellKnownConfig can be null in some cases. In general we should treat that it doesn't exist for some reason and just do nothing. Method was updated.

**Review by ulfcarlin — APPROVED** [review]

**Pull request merged**